doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -197,6 +197,7 @@ nRF5340 Audio
   * Removed the LE Audio controller for nRF5340 library.
     The only supported controller for LE Audio is :ref:`ug_ble_controller_softdevice`.
     This enables use of standard tools for building, configuring, and DFU.
+  * Using low latency configuration as default setting for the nRF5340 Audio application.
```

Comment:
Suggesting that you organize the bullets under nRF5340 Audio as follows:

* Removed:

  * The LE Audio...

* Updated:

  * Low latency configuration to be used as default setting for the nRF5340 Audio application.